src/Bridges/Objective/bridge.jl

```diff
@@ -0,0 +1,118 @@
+                          func::MOI.AbstractScalarFunction)
+    throw(MOI.UnsupportedAttribute(MOI.ObjectiveFunction{typeof(func)}()))
+end
+
+"""
+    function MOI.set(model::MOI.ModelLike, attr::MOI.ObjectiveSense,
+                     bridge::AbstractBridge, sense::MOI.ObjectiveSense)
+
+Return the value of the attribute `attr` of the model `model` for the
+variable bridged by `bridge`.
+"""
+function MOI.set(::MOI.ModelLike, ::MOI.ObjectiveSense,
+                 bridge::AbstractBridge, ::MOI.OptimizationSense)
+    throw(ArgumentError(
+        "Objective bridge of type `$(typeof(bridge))` does not support" *
+        " modifying the objective sense. As a workaround, set the sense to" *
+        " `MOI.FEASIBILITY_SENSE` to clear the objective function and" *
+        " bridges."))
+end
+
+"""
+    function MOI.get(model::MOI.ModelLike, attr::MOI.ObjectiveFunction,
+                     bridge::AbstractBridge)
+
+Return the value of the objective function bridged by `bridge` for the model
+`model`.
+"""
+function MOI.get(::MOI.ModelLike, ::MOI.ObjectiveFunction,
+                 bridge::AbstractBridge)
+    throw(ArgumentError(
+        "ObjectiveFunction bridge of type `$(typeof(bridge))` does not" *
+        " support getting the objective function."))
+end
+
+"""
+    supports_objective_function(
+        BT::Type{<:AbstractBridge},
+        F::Type{<:MOI.AbstractScalarFunction})::Bool
+
+Return a `Bool` indicating whether the bridges of type `BT` support bridging
+objective functions of type `F`.
+"""
+function supports_objective_function(
+    ::Type{<:AbstractBridge}, ::Type{<:MOI.AbstractScalarFunction})
+    return false
+end
+
+"""
+    added_constrained_variable_types(BT::Type{<:MOI.Bridges.Objective.AbstractBridge},
+                                     F::Type{<:MOI.AbstractScalarFunction})
+
+Return a list of the types of constrained variables that bridges of type `BT`
+add for bridging objective functions of type `F`. This fallbacks to
+`added_constrained_variable_types(concrete_bridge_type(BT, F))`
+so bridges should not implement this method.
+```
+"""
+function MOIB.added_constrained_variable_types(
+    BT::Type{<:AbstractBridge}, F::Type{<:MOI.AbstractScalarFunction})
+    return MOIB.added_constrained_variable_types(concrete_bridge_type(BT, F))
+end
+
+"""
+    added_constraint_types(BT::Type{<:MOI.Bridges.Objective.AbstractBridge},
+                           F::Type{<:MOI.AbstractScalarFunction})
+
+Return a list of the types of constraints that bridges of type `BT` add
+add for bridging objective functions of type `F`. This fallbacks to
+`added_constraint_types(concrete_bridge_type(BT, S))`
+so bridges should not implement this method.
+"""
+function MOIB.added_constraint_types(
+    BT::Type{<:AbstractBridge}, F::Type{<:MOI.AbstractScalarFunction})
+    return MOIB.added_constraint_types(concrete_bridge_type(BT, F))
+end
+
+function MOIB.set_objective_function_type(
```

Comment:
Missing a docstring similar to the ones above?

Reply:
It was added in https://github.com/JuliaOpt/MathOptInterface.jl/pull/867/